### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by mazzystr**
Fixes #435

### CODE REVIEW

**Comment by mazzystr** [comment]
@phoracek Petr, would you mind helping me out with a stamp of approval?  Thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Contributing guidelines
 
-Please follow the same [contributing guide](https://github.com/kubevirt/community/blob/master/contributers/contributing.md) as you would follow for any other pull request made to Kubevirt.
+Please follow the same [contributing guide](https://github.com/kubevirt/community/blob/master/contributors/contributing.md) as you would follow for any other pull request made to Kubevirt.
```
Comment (phoracek) [inline CONTRIBUTING.md:3]: What's the point of having a reference to "Page under construction. Check back soon."?

**Comment by mazzystr** [comment]
There isn't an existing CONTRIBUTING page in user-guide so this is no
gain/loss.

Second we have to consolidate all random repo CONTRIBUTING pages to a
single page point of reference in the community repo.

We're basically following the framework set by Kubernetes.

**Comment by phoracek** [comment]
I still don't understand. If this page is not useful, it should be deleted. If we want to keep it, it should be useful.

We should not follow a framework just for the sake of following it

**Comment by mazzystr** [comment]
We either follow or spend time developing one that meets CNCF due diligence
reqs for Incubation graduation.

One way or another we have to get started cleaning up our playground.

**Review by phoracek — APPROVED** [review]

**Pull request merged**